ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -151,7 +151,12 @@ Item {
                   cursorShape: Qt.PointingHandCursor
                   anchors.fill: parent
                   onClicked: {
-                    chatsModel.requestMoreMessages(Constants.fetchRangeLast24Hours)
+                    if(chatsModel.activeChannel.isTimelineChat){
+                        chatsModel.requestMoreTimelineMessages(Constants.fetchRangeLast24Hours);
```

Comment:
I wonder if it shouldn't be the responsibility of the view to check the type of chat and then decide which function to call? I guess it doesn't change much, but if we ever need to call this at multiple places in QML, it would save a few lines.